lib/jellyfish/room.ex

```diff
@@ -281,9 +281,9 @@ defmodule Jellyfish.Room do
         Logger.warning("Unable to add component: incompatible codec")
         {:reply, {:error, :incompatible_codec}, state}
 
-      {:error, :reached_components_limit_hls} ->
+      {:error, :reached_components_limit} ->
         Logger.warning("Unable to add component: reached components limit")
-        {:reply, {:error, :reached_components_limit_hls}, state}
+        {:reply, {:error, :reached_components_limit}, state}
```

Comment:
I think we should mention which component type limit we reached.

Reply:
Don't you think that you should know based on the request you did

Reply:
My mistake it's internal log